config/yugabyte/regression_pipelines/orderby_workloads/postgres/ORDG7_orderby_on_hash_partitioned_table.yaml

```diff
@@ -41,14 +41,22 @@ microbenchmark:
+                    - name: col_bigint_card3_
```

Comment:
Can we use RandomNumber utility , giving range [ 1000001, 1001000 ]

Reply:
Duplicate comment.